Can't resolve './postMock.html

hello, I have a question ,can you help me?


**ERROR in ./node_modules/react-native-web-webview/dist/index.js
Module not found: Error: Can't resolve './postMock.html' in '...'**

I  have already set rules at webpack, but do not works.
`const rule = {
  test: /postMock.html$/,
  use: {
    loader: 'file-loader',
    options: {
      name: '[name].[ext]',
    },
  },
};`

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
   ],
   "scripts": {
     "prepublish": "yarn build",
-    "build": "mkdir -p dist && babel src/index.js --out-file dist/index.js",
+    "build": "mkdir -p dist && babel src --out-dir dist --copy-files",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "babel": {

